uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/form/components/form-sections/leave-type-wizard-form-settings.html

```diff
@@ -0,0 +1,52 @@
+<div class="row">
+  <div class="col-md-3">
+    <ul class="nav nav-pills nav-stacked">
+      <li ng-repeat="settingsTab in form.settingsTabs"
+        ng-class="{ active: settingsTab.active }"
+        ng-click="form.openSettingsTab(settingsTab.name)">
+        <a href="#">
+          {{settingsTab.label}}
+        </a>
+      </li>
+    </ul>
+  </div>
+  <div class="col-md-9">
+    <div class="row" ng-repeat="field in form.getSettingsTabFields()">
+      <div class="col-md-3">
+        <label>
+          {{field.label}}
+          <help-text ng-if="field.helpText">
+            <p class="respect-line-breaks">{{field.helpText}}</p>
+          </help-text>
+        </label>
+      </div>
+      <div class="col-md-9">
+        [template of the field]
+      </div>
+    </div>
+    <hr/>
+    <div class="row">
+      <div class="col-md-12">
+        <div class="btn btn-secondary pull-right"
```

Comment:
use `<button>` given that they are buttons

Reply:
Done.